draft-ietf-quic-transport.md

```diff
@@ -244,8 +245,9 @@ x (*) ...:
 # Streams {#streams}
 
 Streams in QUIC provide a lightweight, ordered byte-stream abstraction to an
```

Comment:
```suggestion
Streams in QUIC provide a lightweight, ordered reliable byte-stream abstraction to an
```

Reply:
I'm not sure I want to make that change and I'm not changing that sentence at the moment, so if you would like to change it, I'd suggest a separate PR.